Tests/ConfigTests/ConfigurationPropertyTests.swift

```diff
@@ -347,6 +347,22 @@ class ConfigurationPropertyTests: XCTestCase {
         expect(actualValue).to(equal(expectedValue))
     }
 
+    func testItCanWriteAnEnvironmentVariableProperty() throws {
+        // Set an environment variable using the libc API
+        setenv("SOMETHING", "testValue", 1)
+        let property = ConfigurationProperty<String>(key: "test", typeHint: "EnvironmentVariable", dict: ["defaultValue": "SOMETHING"])
+        let expectedValue = ##"    static let test: String = #"testValue"#"##
+        let actualValue = try whenTheDeclarationIsWritten(for: property)
+        expect(actualValue).to(equal(expectedValue))
+    }
+
+    func testItWritesAnEmptyStringForAnEnvironmentVariableThatIsNotSet() throws {
+        let property = ConfigurationProperty<String>(key: "test", typeHint: "EnvironmentVariable", dict: ["defaultValue": "DOESNT_EXIST"])
+        let expectedValue = ##"    static let test: String = """##
```

Comment:
I wonder whether we should have some signpost behaviour here. Maybe it should also generate a comment or maybe a warning that the variable isn't set?

Reply:
Fatal error behaviour: https://github.com/dhardiman/Config/pull/9/commits/18000d800f9adef64dd54fd468fae48bee5f5cdc